jetson/percep/pcl.cpp

```diff
@@ -455,17 +456,20 @@ shared_ptr<pcl::visualization::PCLVisualizer> PCL::createRGBVisualizer() {
 /* --- Main --- */
 //This is the main point cloud processing function
 //It returns the bearing the rover should traverse
+//For the PassThroughFilter function we can trust the ZED depth for up to 7000 mm (7 m) for "z" axis. 
+//3000 mm (3m) for "x" is a placeholder, we will chnage this value based on further testing.
 //This function is called in main.cpp
 obstacle_return PCL::pcl_obstacle_detection(shared_ptr<pcl::visualization::PCLVisualizer> viewer) {
     obstacle_return result;
-    PassThroughFilter();
+    PassThroughFilter("z", 7000.0);
+    PassThroughFilter("y", 3000.0);
     DownsampleVoxelFilter();
     RANSACSegmentation("remove");
     std::vector<pcl::PointIndices> cluster_indices;
     CPUEuclidianClusterExtraction(cluster_indices);
     std::vector<std::vector<int>> interest_points(cluster_indices.size(), vector<int> (4));
     FindInterestPoints(cluster_indices, interest_points);
-    bearing = FindClearPath(interest_points, viewer);  
+    bearing = FindClearPath(interest_points, viewer); 
```

Comment:
In the summary of the "general cleanup" include an item for "remove trailing whitespace" and "ensure all files have a single newline at the end"

Reply:
https://github.com/umrover/mrover-workspace/issues/516

Reply:
Y axis filter testing issue: https://github.com/umrover/mrover-workspace/issues/517